### PR TITLE
prisma-utils -> biome

### PR DESCRIPTION
**Pull request by CarlosGamero**
## Changes

Migrating Prisma-utils to use biome

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/prisma-utils/package.json [hunk 1]
```diff
@@ -1,67 +1,64 @@
...
+    "cross-env": "^7.0.3",
+    "dotenv-cli": "^7.4.1",
+    "prettier": "^3.3.2",
+    "prisma": "^5.14.0",
+    "rimraf": "^5.0.7",
+    "typescript": "5.5.3",
+    "vite": "^5.3.3",
+    "vitest": "^1.6.0"
+  },
+  "prettier": "@lokalise/prettier-config"
```
Comment (kibertoad) [inline packages/app/prisma-utils/package.json:63]: no longer needed

File: packages/app/prisma-utils/package.json [hunk 2]
```diff
@@ -1,67 +1,64 @@
...
+  "devDependencies": {
+    "@biomejs/biome": "^1.8.3",
+    "@lokalise/biome-config": "^1.2.2",
+    "@lokalise/package-vite-config": "*",
+    "@lokalise/prettier-config": "*",
+    "@prisma/client": "^5.14.0",
+    "@vitest/coverage-v8": "^1.6.0",
+    "cross-env": "^7.0.3",
+    "dotenv-cli": "^7.4.1",
+    "prettier": "^3.3.2",
```
Comment (kibertoad) [inline packages/app/prisma-utils/package.json:56]: also no longer needed

File: packages/app/prisma-utils/package.json [hunk 3]
```diff
@@ -1,67 +1,64 @@
...
+  },
+  "peerDependencies": {
+    "@prisma/client": "^5.14.0",
+    "prisma": "^5.14.0"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "^1.8.3",
+    "@lokalise/biome-config": "^1.2.2",
+    "@lokalise/package-vite-config": "*",
+    "@lokalise/prettier-config": "*",
```
Comment (kibertoad) [inline packages/app/prisma-utils/package.json:51]: same

File: packages/app/prisma-utils/package.json [hunk 4]
```diff
@@ -1,67 +1,64 @@
...
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    }
+  },
+  "scripts": {
+    "build": "rimraf dist && npm run db:update-client && vite build",
+    "dev": "vite watch",
+    "clean": "rimraf dist .eslintcache",
```
Comment (dariacm) [inline packages/app/prisma-utils/package.json:25]: `eslintcache` can be removed

File: packages/app/prisma-utils/vite.config.ts [hunk 5]
```diff
@@ -6,6 +6,6 @@ import packageJson from './package.json'
 
 /* eslint-disable import/no-default-export */
```
Comment (dariacm) [inline packages/app/prisma-utils/vite.config.ts:7]: We need to replace `eslint` suppression comments with Biome ones (or none if Biome doesn't complain)

File: packages/app/prisma-utils/package.json (hunk 1 above)
Comment (CarlosGamero) [reply packages/app/prisma-utils/package.json]: Good catch! thanks 🙏

File: packages/app/prisma-utils/package.json (hunk 2 above)
Comment (CarlosGamero) [reply packages/app/prisma-utils/package.json]: Solved, thanks 🙏

File: packages/app/prisma-utils/package.json (hunk 4 above)
Comment (CarlosGamero) [reply packages/app/prisma-utils/package.json]: Good catch! thank youu! 🙇

File: packages/app/prisma-utils/biome.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "../../../node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["../../../node_modules/@lokalise/biome-config/configs/biome-base.jsonc"]
```
Comment (CarlosGamero) [inline packages/app/prisma-utils/biome.json:3]: @kibertoad I guess on package we want to have our biome base config right?

Comment (kibertoad) [reply packages/app/prisma-utils/biome.json:3]: yup!

**Review by dariacm — APPROVED** [review]

**Pull request merged**